modules/src/clients/ics07_tendermint/client_state.rs

```diff
@@ -52,6 +51,7 @@ impl ClientState {
         max_clock_drift: Duration,
         latest_height: Height,
         frozen_height: Height,
+        proof_specs: Vec<ibc_proto::ics23::ProofSpec>,
```

Comment:
Should we instead write our own impls of `Serialize/Deserialize` for `modules/src/core/ics23_commitment/specs.rs::ProofSpecs` (i.e. our domain type) and use that instead of the proto types?

Reply:
Yeah I think we should have domain types for all types which we expose publicly somehow, either via the API or to users in the config.

Reply:
https://github.com/informalsystems/ibc-rs/pull/1574/commits/3c42a9e9a6e3a41663a58e3de391e21003697f4d